samples/server/petstore/spring-mvc-j8-async/src/main/java/io/swagger/api/FakeApi.java

```diff
@@ -82,7 +82,7 @@
         if(getObjectMapper().isPresent() && getAcceptHeader().isPresent()) {
             if (getAcceptHeader().get().contains("application/json")) {
                 try {
-                    return CompletableFuture.completedFuture(new ResponseEntity<>(getObjectMapper().get().readValue("{  \"my_string\" : { },  \"my_number\" : { },  \"my_boolean\" : { }}", OuterComposite.class), HttpStatus.NOT_IMPLEMENTED));
+                    return CompletableFuture.completedFuture(new ResponseEntity<>(getObjectMapper().get().readValue("{  \"my_number\" : { },  \"my_string\" : { },  \"my_boolean\" : { }}", OuterComposite.class), HttpStatus.NOT_IMPLEMENTED));
```

Comment:
Arf ! We need to order the fields in the examples or PRs are going to be messy !